src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -77,10 +77,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     String user = userService.getCurrentUser().getEmail();
     String text = Jsoup.clean(request.getParameter("text"), Whitelist.none());
+    //get the value of query parameter
```

Comment:
This comment doesn't add much to the following line.

Reply:
I removed this comment.